Gemfile

```diff
@@ -15,10 +15,20 @@ gem 'uglifier', '>= 1.3.0'
 # See https://github.com/rails/execjs#readme for more supported runtimes
 # gem 'therubyracer', platforms: :ruby
 
+gem 'sidetiq'
+gem 'github_api'
+gem 'sinatra', :require => nil
```

Comment:
sinatra is needed for the sidekiq monitoring

https://github.com/mperham/sidekiq/wiki/Monitoring